server.Dockerfile

```diff
@@ -3,8 +3,10 @@ MAINTAINER David Lawrence "david.lawrence@docker.com"
 
 RUN apk add --update git gcc libc-dev && rm -rf /var/cache/apk/*
 
-# Install SQL DB migration tool
-RUN go get github.com/mattes/migrate
+# Pin to the specific v1 version
+RUN git clone -b v1 https://github.com/mattes/migrate.git /go/src/github.com/mattes/migrate/
+RUN go get -u -v github.com/mattes/migrate && \
```

Comment:
Fewer layers (for all that it matters in the weeks until we get 17.05 multi-stage builds):
```
RUN git clone ... \
    && go get ...
```
Also, not sure we need the `-u` flag. If we just cloned, it shoudn't be necessary to ask go to also do a network update.